x/community/types/errors.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var (
+	ErrInvalidProposal      = sdkerrors.Register(ModuleName, 1, "invalid community pool proposal")
```

Comment:
suggest additional errors for cases where authority is invalid or message to execute is not in the whitelist